src/components/router/router.tsx

```diff
@@ -8,13 +8,16 @@ import TreeDetail from "../tree-detail/tree-detail";
 import { useLocationEventListener } from "./hooks/use-location-event-listener";
 import { useUrlState } from "./store";
 import { PasswordReset } from "../profile/profile-logged-in/password-reset";
+import Filter from "../filter/filter";
 
 const Router: React.FC = () => {
 	const url = useUrlState((state) => state.url);
 	const setPathname = useUrlState((state) => state.setPathname);
 	const treeId = url.searchParams.get("treeId");
 	useLocationEventListener();
 
+	const [showFilter, setShowFilter] = useState(false);
```

Comment:
same comment regarding the naming of the boolean than above, and also I'd suggest to use a zutand store for this as this state is shared among multiple components.

Reply:
maybe in the existing `filterStore`?

Reply:
moved to filter store